content/en/monitors/service_level_objectives/monitor.md

```diff
@@ -6,57 +6,75 @@ further_reading:
 
-For example, if you have a Metric Monitor that is configured to alert when user request latency is greater than 250ms, you could set a monitor-based SLO on that monitor. Let’s say you choose an SLO target of 99% over the past 30 days. What this means is that the latency of user requests should be less than 250ms 99% of the time over the past 30 days. To set this up, you would:
+### Set your SLO targets
 
-1. Select a single monitor or,
-2. Select multiple monitors (up to 20) or,
-3. Select a single [multi-alert monitor][5] and select specific monitor groups (up to 20) to be included in SLO calculation using the **Calculate on selected groups** toggle.
+Select a **target** percentage, **time window**, and optional **warning** level.
 
-**Supported Monitor Types:**
+The target percentage specifies the portion of time the underlying monitor(s) of the SLO should be in an OK state.
 
-- Metric Monitor Types (Metric, Integration, APM Metric, Anomaly, Forecast, Outlier)
-- Synthetic
-- Service Checks (open beta)
+The time window specifies the rolling time period over which the SLO runs its calculation.
 
-**Example:** You might be tracking the uptime of a physical device. You have already configured a metric monitor on `host:foo` using a custom metric. This monitor might also ping your on-call team if it's no longer reachable. To avoid burnout you want to track how often this host is down.
+Depending on the value of the SLI, the Datadog UI displays the SLO status in a different color:
+- While the SLI remains above the target, the UI displays the SLO status in green.
+- When the SLI falls below the target, the UI displays the SLO status in red.
+- If you included a warning level, and the SLI falls above the warning but below the target level, the UI displays the SLO status in yellow.
 
-### Set your SLO targets
+The time window you choose changes the available precision for your monitor-based SLOs:
+- 7-day and 30-day time windows allow up to two decimal places.
+- 90-day time windows allow up to three decimal places.
+
+In the details UI for the SLO, Datadog displays two decimal places for SLOs configured with 7-day and 30-day time windows, and three decimal places for SLOs configured with 90-day time windows.
 
-An SLO target is comprised of the target percentage and the time window. When you set a target for a monitor-based SLO the target percentage specifies what portion of the time the underlying monitor(s) of the SLO should be in an OK state, while the time window specifies the rolling time period over which the target should be tracked.
+A 99.999% target for a 7-day or 30-day time window results in an error budget of 6 seconds or 26 seconds, respectively. Monitors evaluate every minute, so the granularity of a monitor-based SLO is also 1 minute. Therefore, one alert would fully consume and overspend the 6 second or 26 second error budget in the previous example. In practice, teams cannot satisfy such small error budgets.
```

Comment:
We should make it clearer that the point of this example is to illustrate why three decimal places don't make sense for 7-day or 30-day targets